config/tokensource.go

```diff
@@ -19,6 +19,13 @@ import (
 
 type sourceType = string
 
+const (
+	// APIAudienceID is the API identifier configured in the auth provider and
+	// must be provided when requesting an access token for the API. The value
+	// is the same regardless of environment
+	APIAudienceID = "https://api.hashicorp.cloud"
```

Comment:
I am not sure if this will also be the case for the EU. @alexmunda do you know?

Reply:
I don't think we need to/will change the token audience as it is the same in every environment today, so this should be fine. @ohm correct me if you think otherwise.

Reply:
I'll leave it as-is then. Thanks Gents